cocotb/scheduler.py

```diff
@@ -184,14 +186,14 @@ class Scheduler(object):
         - Any pending writes are cached and do not happen immediately
 
     ReadOnly mode
-        - Corresponds to cbReadOnlySynch (VPI) or vhpiCbLastKnownDeltaCycle
+        - Corresponds to :any:`cbReadOnlySynch` (VPI) or :any:`vhpiCbLastKnownDeltaCycle`
           (VHPI).  In this state we are not allowed to perform writes.
 
     Write mode
-        - Corresponds to cbReadWriteSynch (VPI) or vhpiCbEndOfProcesses (VHPI)
+        - Corresponds to :any:`cbReadWriteSynch` (VPI) or :c:macro:`vhpiCbEndOfProcesses` (VHPI)
           In this mode we play back all the cached write updates.
 
-    We can legally transition from normal->write by registering a ReadWrite
+    We can legally transition from Normal→Write by registering a :class:`~cocotb.triggers.ReadWrite`
```

Comment:
Not sure making this file contain unicode characters is a particularly good idea, especially since we still support python2

```suggestion
    We can legally transition from Normal to Write by registering a :class:`~cocotb.triggers.ReadWrite`
```

Reply:
Right; done.